hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/overview/overview.tsx

```diff
@@ -320,6 +327,10 @@ export class Overview extends React.Component<Record<string, object>, IOverviewS
               <OverviewCard title="OM Service" loading={loading} data={omServiceId} icon='file-text' linkToUrl='/Om' />
             </Col>
           }
+          <Col xs={24} sm={18} md={12} lg={12} xl={6}>
+            <OverviewCard loading={loading} title='Decommissioning Datanodes Summary' data={decommissionInfoCount.toString()} icon='hourglass'
```

Comment:
Since icons are not supported like this in AntD v4, we need to specify the icon in OverviewCard component as well.
Check the following function and add the hourglass optionn in that:
https://github.com/apache/ozone/blob/975a8d85a343a3b7a971b6798493280684456136/hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/components/overviewCard/overviewCard.tsx#L64

Reply:
@devabhishekpal 
I merged code with latest master I could not see const IconSelector = ({ iconType, ...extras }: { iconType: string }) => { 

Can we do in seprate pr or we can have discussion when to add changes